config/initializers/secret_token.rb

```diff
@@ -5,3 +5,11 @@
 # Make sure the secret is at least 30 characters and all random,
 # no regular words or you'll be exposed to dictionary attacks.
 Huginn::Application.config.secret_key_base = ENV['APP_SECRET_TOKEN']
+
+# Your encryption key for encrypting sensitive data in the database, such as
```

Comment:
This should probably be in a different initializer, or `ENV['APP_ENCRYPTION_PASSPHRASE']` can be referenced in app/models/user_credential.rb so that this isn't needed.